src/Tools/dotnet-monitor/Egress/AzureBlob/AzureBlobEgressProvider.cs

```diff
@@ -117,7 +132,20 @@ public AzureBlobEgressProvider(ILogger<AzureBlobEgressProvider> logger)
             }
         }
 
-        private Uri GetAccountUri(AzureBlobEgressProviderOptions options, out string accountName)
+        private bool CheckQueueEgressOptions(AzureBlobEgressProviderOptions options)
+        {
+            bool queueNameSet = !string.IsNullOrEmpty(options.QueueName);
+            bool queueAccountUriSet = null != options.QueueAccountUri;
+
+            if ((queueNameSet && !queueAccountUriSet) || (!queueNameSet && queueAccountUriSet))
```

Comment:
```suggestion
            if (queueNameSet ^ queueAccountUriSet)
```

XOR - one of them is true but not both